Dockerfile

```diff
@@ -38,25 +36,16 @@ RUN git clone https://github.com/genuinetools/img.git "$GOPATH/src/github.com/ge
     && cd "$GOPATH/src/github.com/genuinetools/img" \
     && make static && mv img /usr/bin/img
 
-FROM alpine:3.7
-MAINTAINER Ce Gao <gaoce@caicloud.io>
+FROM docker.artifactory-test.corp.linkedin.com/tensorflow/lipy-relevance-image-hdfs:0.1.404
```

Comment:
We cannot get the image from Linkedin's private registry, so I think we cannot update the code here.